doc/monitoring/metrics_reference.rst

```diff
@@ -291,3 +291,71 @@ Those metrics provides CPU usage statistics. Only for Linux.
   ``{kind="user", thread_name="tarantool", thread_pid="pid", file_name="init.lua"}``,
   where ``kind`` is ``user`` or ``system``, ``thread_name`` is ``tarantool``, ``wal``,
   ``iproto`` or ``coio``, ``file_name`` is entrypoint file name, e.g. ``init.lua``.
+
+.. _metrics-vinyl:
+
+-------------------------------------------------------------------------------
+Vinyl
+-------------------------------------------------------------------------------
+
+Vinyl metrics provide :ref:`vinyl engine <storing-data-with-vinyl>` statistics.
+
+**Disk**
+
+* ``tnt_vinyl_disk_data_size`` - the amount of data that has gone into files, in bytes
+
+* ``tnt_vinyl_disk_index_size`` - the amount of index that has gone into files, in bytes
+
+* ``tnt_vinyl_disk_data_compacted_size`` - sum size of data stored at the last LSM tree level,
```

Comment:
Is it 'raw data' size? I mean that we should use more obvious descriptions if we could do that.